.prettierignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Doesn't Prettier ignore `node_modules` by default?

Reply:
Yes, it does. I just add 'cuz reasons.

Should I remove it?

Reply:
Yeah